playground/recursive-set.nix

```diff
@@ -0,0 +1,6 @@
+rec {
+    foo.bar = 13;
```

Comment:
WHen I hover over the `13` here, I don't seem to get any result.

Reply:
Weird. Would you mind trying this again (along with the attr-set hovering comment below) with the new PR code?

Reply:
Works now.